CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
-# 0.1.0
+# 0.2.0 (2024-11-10)
```

Comment:
```suggestion
# 0.2.0 (2024-12-10)
```

Reply:
This looks great! I don't see anything else aside from keeping this up to date with any other changes that get merged.

Reply:
Updated the docs now.